resources/svc/default-mysql.yml

```diff
@@ -1,18 +1,17 @@
-version: '3.1'
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.7'
 
 services:
   mysql:
-    image: mysql
-    restart: always
+    image: mysql:8.0.26
+    # If you want to relocate the mysql data (volume) to your local directory, uncomment the lines below
+    # volumes:
+    #   - ~/volumes/entando-bundle-cli/%BUNDLENAME%/mysql/:/var/lib/mysql/
     environment:
-      - MYSQL_DATABASE=%BUNDLENAME%-db
-      - MYSQL_ROOT_PASSWORD=adminadmin
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
```

Comment:
Empty password? Let's use `%BUNDLENAME%-user` and `%BUNDLENAME%-password`, same for postgres

Reply:
done. I added MYSQL_RANDOM_ROOT_PASSWORD as root password is needed in order the service to work